src/io/flutter/run/daemon/FlutterDaemonController.java

```diff
@@ -116,7 +114,7 @@ void removeDeviceId(String deviceId) {
   public void forkProcess(Project project) throws ExecutionException {
     try {
       GeneralCommandLine commandLine = createCommandLine(project);
-      myHandler = new OSProcessHandler(commandLine);
+      myHandler = new DaemonProcessWrapper(new OSProcessHandler(commandLine));
```

Comment:
It looks like both startDeviceController and startApp call forkProcess, and then the result is stored in an instance variable. Is a `FlutterDaemonController` shared between multiple running apps?

I'm woefully unfamiliar with the architecture here, and why we're wrapping a process here, instead of creating a new DaemonProcess style subclass of ProcessHandler.